miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditLibraryController.java

```diff
@@ -822,82 +764,19 @@ public ModelAndView editPropagateSamples(@PathVariable String sampleIds, ModelMa
         } else {
           hasPlain = true;
         }
-        LibraryDto library = isDetailedSampleEnabled() ? new DetailedLibraryDto() : new LibraryDto();
-        if (namingScheme.hasLibraryAliasGenerator()) {
```

Comment:
Some of this needs to be kept, as EI wants to see the bulk propagate page with the library aliases already filled in.